osf/models/node.py

```diff
@@ -2409,13 +2409,44 @@ def _add_related_source_tags(self, contributor):
         ).first() or osf_provider_tag
         contributor.add_system_tag(source_tag)
 
-    def _remove_from_associated_collections(self):
+    def _remove_from_associated_collections(self, auth=None, force=False):
         for submission in self.guids.first().collectionsubmission_set.all():
             associated_collection = submission.collection
             if associated_collection.is_bookmark_collection and not self.deleted:
                 if self.contributors.filter(pk=associated_collection.creator.id).exists():
                     continue
-            submission.delete()
+
+            if submission.state not in [CollectionSubmissionStates.REMOVED, CollectionSubmissionStates.REJECTED]:
+                user = getattr(auth, 'user', None)
+
+                if submission.state == CollectionSubmissionStates.ACCEPTED:
+                    submission.remove(
+                        user=user,
+                        comment='Removed from collection due to implicit removal due to privacy changes.',
+                        removed_due_to_privacy=True
```

Comment:
This is inconsistent with the other kwargs (looks like the email template is keying on this term, not `force`)

Reply:
A that is necessary to indicate an different email template is used. So it is inconsistent, but it's difficult to pass the info otherwise. Potentially there are better ways.